build.go

```diff
@@ -935,9 +936,12 @@ func NewGometalinter(linter string, dirs ...string) *gometalinter {
 }
 
 func (g *gometalinter) verifyInstalled() {
-	if _, err := runError(g.command, "--disable-all"); err != nil {
-		log.Fatalf("gometalinter is not installed")
+	if !g.verifiedInstallation {
+		if _, err := runError(g.command, "--disable-all"); err != nil {
+			log.Fatalf("gometalinter is not installed")
+		}
 	}
+	g.verifiedInstallation = true
```

Comment:
This will still verify it multiple times, as the methods are no longer pointer receivers, and you have multiple instances of the objects anyway.